src/interface/graph.thrift

```diff
@@ -25,6 +25,8 @@ enum ErrorCode {
 
     E_SYNTAX_ERROR = -7,
     E_EXECUTION_ERROR = -8,
+    // Nothing is executed When command is comment
```

Comment:
what is the comment style?  // should the comment be placed after the error code in the same line?